import_models_to_verta/mlflow_databricks_bulk_import_example.ipynb

```diff
@@ -0,0 +1,341 @@
+    "mlflow_models += result.to_list()\n",
+    "while result.token:  # handle paginated results\n",
+    "    result: PagedList = mlflow_client.search_registered_models(\n",
+    "    page_token=result.token\n",
+    "    )\n",
+    "    mlflow_models += result.to_list()\n",
+    "\n",
+    "mlflow_models.sort(key=lambda x: x.name)\n",
+    "for mv in mlflow_models:\n",
+    "    print(f\"name={mv.name}; version={mv.version}\")"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Reduce the list if desired"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# Example: filter out models with \"test\" in the name\n",
+    "models_for_import = [model for model in mlflow_models if \"test\" not in model.name]"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Import Models to Verta"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# These are the names of files that Verta uses internally and cannot be overwritten\n",
+    "protected_file_names = [\n",
+    "    \"custom_modules\",\n",
+    "    \"model\",\n",
+    "    \"model.pkl\",\n",
+    "    \"model_api.json\",\n",
+    "    \"requirements.txt\",\n",
+    "    \"train_data\",\n",
+    "    \"tf_saved_model\",\n",
+    "    \"setup_script\",\n",
+    "]"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "failed_model_imports: List[MLflow_RegisteredModel] = list()\n",
+    "successful_model_imports: List[MLflow_RegisteredModel] = list()\n",
+    "model_messages: Dict[MLflow_RegisteredModel, List[str]] = dict()\n",
+    "\n",
+    "for rm in models_for_import:\n",
+    "    model_messages[rm] = list()\n",
+    "\n",
+    "    try:\n",
+    "        verta_rm = verta_client.create_registered_model(\n",
+    "            name=rm.name,\n",
+    "            desc=rm.description,\n",
+    "            # Note: hacking into labels because we don't have a way to add attributes to registered models\n",
+    "            labels=[\n",
+    "                \"mlflow_import\",\n",
+    "                \"mlflow_creation_time:\" + str(rm.creation_timestamp),\n",
+    "                \"mlflow_last_updated_time:\" + str(rm.last_updated_timestamp),\n",
+    "                \"mlflow_tags:\" + \",\".join(rm.tags),\n",
+    "            ],\n",
+    "        )\n",
+    "        model_messages[rm].append(f\"created new registered model in Verta for {rm.name}\")\n",
+    "    except ValueError:\n",
+    "        model_messages[rm].append(f\"a registered model named \\\"{rm.name}\\\" already exists in Verta. Skipping import.\")\n",
```

Comment:
I don't think this is always true. A `ValueError` can be raised for all sorts of conditions, not just an RM already existing.